src/components/Login/LoginForm/index.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import styles from './loginForm.module.scss';
+import LoginCaption from './LoginCaption';
+import LoginButton from './LoginButton';
+import Input from '../../shared/Input';
+import { getPassword, getUsername } from '../../../store/Login/selectors';
+import { changePasswordValue, changeUsernameValue, formSubmit } from '../../../store/Login/actions';
+import AlertMessage from '../../shared/AlertMessage';
+import { preventDefaultHandler } from '../../../utils';
+
+const ERROR_MESSAGE = 'Wrong username or password. Please, try again';
+
+const LoginForm = ({
+  username, password, changePassword, changeUsername, submitForm
+}) => (
+  <div className={styles.loginForm}>
+    <LoginCaption />
+    <div className={styles.line} />
+    <form onSubmit={preventDefaultHandler(submitForm)}>
```

Comment:
It's not the best idea to pass function call in a handler, cause every time render is called new function creates
It's better to create some link, like const handler = preventDefaultHandler(submitForm)
Or call it in mapDispatchToProps maybe?
Or in actions?

Reply:
Fixed